api/urls.py

```diff
@@ -0,0 +1,27 @@
+from django import VERSION as DJANGO_VERSION
+if DJANGO_VERSION[:2] >= (1, 4):
+    from django.conf.urls import (
+        patterns,
+        url,
+        include
+    )
+else:
+    from django.conf.urls.defaults import (
+        patterns,
+        url,
+        include
+    )
+from rest_framework import routers
+from viewsets import HostViewSet
+
+router = routers.DefaultRouter()
+router.register(r'hosts', HostViewSet)
+
+
+urlpatterns = patterns(
```

Comment:
not sure about this patterns call here. I remember having problems with doubling calling this (it's already called from the main urls.py). Let me get back to you on this one after some testing